### ISSUE TITLE
 expected `*const u8`,     found `*const i8`

### ISSUE THREAD

**Issue by andresv** [tomaka/shared_library#3]
Hi,

I cannot compile this lib with rust-1.5-stable on RaspberryPi 2.

```
   Compiling shared_library v0.1.2
/home/pi/.multirust/toolchains/1.5/cargo/registry/src/github.com-48ad6e4054423464/shared_library-0.1.2/src/dynamic_library.rs:112:37: 112:56 error: mismatched types:
 expected `*const u8`,
    found `*const i8`
(expected u8,
    found i8) [E0308]
```

**Comment by andresv** [comment]
Fixed with 0.1.3: https://github.com/tomaka/shared_library/commit/2d86a5affe5e797aafbed177abe255146383fefc

**Comment by tomaka** [comment]
Ah sorry, forgot to reference the issue.
